src/mistune/__init__.py

```diff
@@ -8,16 +8,26 @@
     Documentation: https://mistune.lepture.com/
 """
 
-from .markdown import Markdown
-from .core import BlockState, InlineState, BaseRenderer
+from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, Union
+
+from typing_extensions import Literal
```

Comment:
Why not use `typing.Literal`?

Reply:
Python 3.7 doesn't have `typing.Literal`, that got introduced in Python 3.8

https://typing-extensions.readthedocs.io/en/latest/#Literal